### DISCUSSION TITLE
I no longer have my license codes

### DISCUSSION THREAD

**Discussion by SKP-bz69** [aclap-dev/video-downloadhelper#1560] [License & Purchase requests]
### What's your issue?

hello, following the reset of my PC I need my license codes, which I unfortunately did not save. 
My email : toniomarie0297@gmail.com
Thank you in advance, sorry for the inconvenience caused

### The 10 first digits (only 10) of your license number or email address

toniomarie0297@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
I sent you an email.
